src/olympia/templates/impala/base.html

```diff
@@ -24,9 +24,6 @@
     {% block site_css %}
       {{ css('zamboni/impala') }}
       {{ css('restyle/css') }}
-      <!--[if lt IE 9]>
-        <script src="{{ static('js/lib/iepp.min.js') }}"></script>
-      <![endif]-->
       <!--[if lte IE 9]>
         <link rel="stylesheet" href="{{ static('css/legacy/ie9.css') }}">
```

Comment:
I presume of all of those IE9 is the highest versionwise? If so we're probably good to remove all of it.